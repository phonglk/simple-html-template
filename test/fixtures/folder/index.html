<div>
  <include src='./header.partial.html' />
  Index
</div>
