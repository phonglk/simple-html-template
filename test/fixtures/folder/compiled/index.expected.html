<div>
  <Title>Header</Title>

  Index
</div>
